web/pgadmin/misc/__init__.py

```diff
@@ -84,11 +83,17 @@ def register_preferences(self):
             options=theme_options,
             control_props={
                 'allowClear': False,
+                'creatable': False,
             },
+        )
+
+        self.preference.register(
```

Comment:
there is no need for a new preference. You could add one more drop-down option in existing themes - `System`

Reply:
Generally, sync with os option is given separately for theme selection for example - pycharm. Providing option this way will be more noticeable than in dropdown.

Reply:
Slack, Chrome has different way of doing it. Even VS code.
The point is - having 2 variables to decide a theme can be more confusing. It should be either one.

<img width="865" alt="image" src="https://github.com/pgadmin-org/pgadmin4/assets/31855755/7974611a-9075-4975-8739-bb8272617055">

<img width="799" alt="image" src="https://github.com/pgadmin-org/pgadmin4/assets/31855755/a5fef9a9-5e19-40fa-a160-cd36808328bc">

Reply:
Firefox. Most apps do not have a separate option.

<img width="778" alt="image" src="https://github.com/pgadmin-org/pgadmin4/assets/31855755/24d6766e-b73c-4f59-9dbd-f4d21c26a5de">

Reply:
Done.